image/src/index.ts

```diff
@@ -73,7 +73,7 @@ app.all('/ipfs/*', async (c) => {
       }
 
       // fallback to cf-ipfs
-      return Response.redirect(`${c.env.CLOUDFLARE_GATEWAY}/ipfs/${path}`, 302);
+      return c.redirect(`${c.env.CLOUDFLARE_GATEWAY}/ipfs/${path}`, 302);
```

Comment:
Are we sure this is the fix? There are total of 5 lines using `Response.redirect`